src/core/flow.test.ts

```diff
@@ -1783,6 +1959,74 @@ test('stage events should not be processed out of turn', () => {
   });
 });
 
+describe('backwards compatibility for moveLimit', () => {
+  test('turn config maps moveLimit to minMoves/maxMoves', () => {
+    const flow = Flow({
+      moves: {
+        noop: () => {},
+      },
+      turn: {
+        moveLimit: 2,
+      },
+    });
+    let state = flow.init({ ctx: flow.ctx(2) } as State);
+
+    expect(state.ctx.turn).toBe(1);
+    expect(state.ctx.currentPlayer).toBe('0');
+
+    state = flow.processMove(state, makeMove('noop', null, '0').payload);
+    state = flow.processMove(state, makeMove('noop', null, '0').payload);
+
+    expect(state.ctx.turn).toBe(2);
+    expect(state.ctx.currentPlayer).toBe('1');
+
+    state = flow.processMove(state, makeMove('noop', null, '1').payload);
+
+    state = flow.processEvent(state, gameEvent('endTurn', null, '1'));
+
+    // player should not be able to endTurn because they haven't made minMoves yet
+
+    expect(state.ctx.turn).toBe(2);
+    expect(state.ctx.currentPlayer).toBe('1');
+  });
+
+  // test('setActivePlayers maps moveLimit to minMoves/maxMoves', () => {
+  //   const flow = Flow({
+  //     moves: {
+  //       setActivePlayers: (G, ctx) => {
+  //         ctx.events.setActivePlayers({ all: 'A', moveLimit: 1 });
+  //       },
+  //     },
+  //   });
+  //   let state = flow.init({ ctx: flow.ctx(2) } as State);
+
+  //   expect(state.ctx.turn).toBe(1);
+  //   expect(state.ctx.currentPlayer).toBe('0');
+
+  //   state = flow.processMove(
+  //     state,
+  //     makeMove('setActivePlayers', null, '0').payload
+  //   );
```

Comment:
The issue here is that testing flow like this doesn’t actually run the moves, it just does the framework stuff around that (incrementing counts etc.) It would be better to use a `processEvent` call here to dispatch the `setActivePlayers` event:

```js
    state = flow.processEvent(
      state,
      gameEvent('setActivePlayers', { all: 'A', moveLimit: 1 })
    );
```

Then you can assert that `_activePlayersMaxMoves` is set as expected and that `_activePlayersMinMoves` remains `null` for `moveLimit`.